walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/index.html

```diff
@@ -10,7 +10,7 @@
     <div id="content">
       <h1>Random Quotes</h1>
       <p>Click here to see a random quote:</p>
-      <button onclick="getRandomQuote()">Fetch a quote!</button>
+      <button onclick="getRandomQuoteUsingArrowFunctions()">Fetch a quote!</button>
```

Comment:
You shouldn't check in changes to the walkthrough files

Reply:
In the walkthrough, it said, "Try changing the onclick attribute of the button to call the getRandomQuoteUsingArrowFunctions() function." Maybe it just meant to try it and change it back....?

Reply:
Making changes to the walkthrough files are fine, it's just preferred if you don't include those changes in your pull requests.